ci-operator/step-registry/openstack-k8s-operators/build/openstack-k8s-operators-build-ref.yaml

```diff
@@ -0,0 +1,14 @@
+ref:
+  as: openstack-k8s-operators-build
+  from: src
+  commands: openstack-k8s-operators-build-commands.sh
+  resources:
+    requests:
+      cpu: 1000m
+      memory: 100Mi
+  credentials:
```

Comment:
This credential doesn't exist anymore. Your test might fail because of that. You can remove since it is not being used in  this ping test.

Reply:
ok